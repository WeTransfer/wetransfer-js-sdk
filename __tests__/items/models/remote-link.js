const { RemoteLink } = require('../../../src/items/models');

describe('RemoteLink model', () => {
  let link = null;
  beforeEach(() => {
    link = new RemoteLink({
      id: 'random-hash',
      content_identifier: 'web_content',
      meta: {
        title: 'WeTransfer',
      },
      url: 'https://wetransfer.com',
    });
  });

  describe('contructor', () => {
    it('should create a normalized link', () => {
      expect(link).toMatchSnapshot();
    });
  });
});
