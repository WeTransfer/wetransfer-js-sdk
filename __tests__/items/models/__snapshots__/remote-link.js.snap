// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoteLink model contructor should create a normalized link 1`] = `
RemoteLink {
  "content_identifier": "web_content",
  "id": "random-hash",
  "meta": Object {
    "title": "WeTransfer",
  },
}
`;
